Dockerfile

```diff
@@ -23,7 +25,7 @@ RUN curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/
 RUN trivy fs --format spdx-json --output /sbom.spdx.json /SBOM
 RUN trivy sbom /sbom.spdx.json --severity UNKNOWN,HIGH,CRITICAL --exit-code 1
 
-FROM node:20-alpine3.17
+FROM $BASE_IMAGE
```

Comment:
Feels like maybe we should use the same base image for the other builder phases above (on lines 3 and 11)?

I don't think binary compatibility will be an issue since those steps are only building for Node and EVM. But if someone used a base image other than Node 20 here, for instance... there could definitely be runtime incompatibility.

Reply:
Going to add a BUILDER_IMAGE here as I think that makes sense